metadata-ingestion/scripts/install_deps.sh

```diff
@@ -2,7 +2,8 @@
 set -euxo pipefail
 
 if [ "$(uname)" == "Darwin" ]; then
-    brew install librdkafka
```

Comment:
Does this not need to be installed anymore?

Reply:
nope - kafka should have prebuilt wheels for m1 macs

you'd only need that if building kafka from source, but I don't expect anyone to need to do that